### PR TITLE
Add build script at repository root

### PR DESCRIPTION
**Pull request by ppebb**
I found myself somewhat annoyed by all of the disparate build scripts in each directory so I wrote one that calls the ones located in each directory.

As far as I'm aware, devkitppc requires msys so the script should be fine for windows users as well.

### CODE REVIEW

File: make.sh [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        echo "$SCRIPT_DIR/misc/private-key-OUT.pem not found! Please generate keys using '--keys' before attempting to build anything else"
+        exit 1
+    fi
+fi
+
+if $payload; then
+    cd "$PAYLOAD_DIR"
+    echo "Making payloads!"
+
+    python make-payload.py -j16
```
Comment (noahpistilli) [inline make.sh]: Probably should make the number of cores used as an argument in the script rather than hardcode 16.

File: make.sh [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+    echo "Copying stage1.bin to dist!"
+    cp "$SCRIPT_DIR/stage1/build/stage1.bin" "$SCRIPT_DIR/dist/stage1.bin"
+fi
+
+if $patch; then
+    cd "$PATCH_DIR"
+    echo -e "Generating patches!"
+
+    python make-patch.py -j16
```
Comment (noahpistilli) [inline make.sh]: Ditto

File: make.sh (hunk 1 above)
Comment (ppebb) [reply make.sh]: Good catch, fixed by allowing additional args for each script to be provided after `--`

**Review by hakusaro — APPROVED** [review]

**Review by chairclr — APPROVED** [review]
LGTM! (let's get this merged)

**Pull request merged**